lib/facterdb.rb

```diff
@@ -137,13 +137,17 @@ def self.valid_filters?(filters)
 
   # @return [Array[Hash[Symbol, Any]]] array of hashes of facts
   # @param filter [Object] The filter to convert to jgrep string
-  def self.get_facts(filter = nil, cache = true)
+  # @param symbolize_keys [Boolean]
+  #   Whether to symbolize the keys. Note this is only on the top level and not
+  #   on nested values.
+  def self.get_facts(filter = nil, cache = true, symbolize_keys = true)
```

Comment:
@ekohl you previously had `symbolize_keys: true`. You want to keep that or should we stick with `symbolize_keys = true`?